src/style.css

```diff
@@ -0,0 +1,75 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(../src/fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+* {
+  font-family: "Avenir", sans-serif;
+}
+
+.search--bar {
+  width: 100%;
+  margin-top: 20px;
+}
+
+.bars--big {
+  height: 70px;
+}
+
+.bars--small {
+  height: 42px;
+}
+
+.search--bars {
+  display: flex;
+  align-items: center;
+  box-sizing: border-box;
```

Comment:
it's better to set for each element on the page (use `*` selector)

Reply:
how do i fix this so my style doesn't break